client/landing/gutenboarding/components/plans/plans-grid/index.tsx

```diff
@@ -67,31 +72,34 @@ const PlansGrid: React.FunctionComponent< Props > = ( { confirmButton, cancelBut
 			</div>
 
 			<div className="plans-grid__details">
-				{ showDetails && (
-					<div className="plans-grid__details-container">
-						<div className="plans-grid__details-heading">
-							<Title>{ __( 'Detailed comparison' ) }</Title>
+				<div className="plans-grid__details-heading">
+					<Title>{ __( 'Detailed comparison' ) }</Title>
+				</div>
+				<div className="plans-grid__details-container">
+					<PlansDetails>
+						<div className="plans-grid__details-actions">
+							{ showDetails ? (
+								<Button
+									className="plans-grid__details-toggle-button is-collapse-button"
+									isLarge
+									onClick={ handleDetailsToggleButtonClick }
+								>
+									<span>{ __( 'Less details' ) } </span>
+									<Icon icon="arrow-up" size={ 20 }></Icon>
+								</Button>
+							) : (
+								<Button
+									className="plans-grid__details-toggle-button is-expand-button"
+									isLarge
+									onClick={ handleDetailsToggleButtonClick }
+								>
```

Comment:
I kinda think feel that the table should be below the Show more/Show Less. This way you can toggle hiding and showing from the same button. Currently, the button disappears (visually) and you'll have to look for it to "show less".

Reply:
I feel the same way too. Created an issue for design to consider. https://github.com/Automattic/wp-calypso/issues/42252